models/spleen_deepedit_annotation/configs/inference.json

```diff
@@ -205,8 +198,16 @@
         "val_handlers": "@handlers",
         "amp": true
     },
+    "checkpointloader": {
+        "_target_": "CheckpointLoader",
+        "load_path": "$@bundle_root + '/models/model.pt'",
+        "load_dict": {
+            "model": "@network"
+        }
+    },
     "initialize": [
-        "$monai.utils.set_determinism(seed=123)"
+        "$monai.utils.set_determinism(seed=123)",
+        "$@checkpointloader(@evaluator)"
```

Comment:
Thanks, I will confirm the weights load time first, and then merge this PR

Reply:
Please also test the TensorRT inference, ensure it's not affected.

Thanks.